src/modules/wallets/sagas.js

```diff
@@ -17,8 +17,8 @@ import blockchains from 'configurables/blockchains'
 
 const checkRegistration = function * ({ blockchainKey, userID }) {
   try {
-    const { exists: isRegistered, pending: isRegistrationPending } = yield call(api.checkRegistration(blockchainKey), { user: userID })
-    return { isRegistered, isRegistrationPending, status: 'registration-checked' }
+    const isRegistered = window.localStorage.getItem(`${blockchainKey}-privateKey-${userID}`)
+    return { isRegistered: !!isRegistered, isRegistrationPending: false, status: 'registration-checked' }
```

Comment:
this should be !!privateKey, and isRegistrationPending: false has no more sense here (setting here always to false makes no sense, should be removed and handled after)